tests/bootstrap.php

```diff
@@ -67,12 +67,12 @@ function xwp_install_edd() {
 
 	edd_install();
 
-	global $current_user, $edd_options;
+	global $edd_options;
 
 	$edd_options = get_option( 'edd_settings' );
 
-	$current_user = new WP_User(1);
```

Comment:
This was setting up the test environment as if an admin user had logged in. It should be the responsibility of each test to setup and tear-down the environment state as necessary.